ansible.cfg

```diff
@@ -4,6 +4,7 @@ inventory = inventory
 #timeout = 60
 force_handlers = True
 stdout_callback = yaml
+allow_world_readable_tmpfiles = True
```

Comment:
what do we need this for?

Reply:
Brought back pipelineing in ansible.cfg, allow_world_readable_tmpfiles not needed anymore